.gitignore

```diff
@@ -123,3 +123,4 @@ Carthage
 .classpath
 .project
 .settings
+*.dep
```

Comment:
Where do these `*.dep` come from?

Reply:
I saw those after opening and build the controller projects with Xcode